src/shared/capture/capture_basler.cpp

```diff
@@ -101,8 +104,20 @@ bool CaptureBasler::_buildCamera() {
 				Pylon::CTlFactory::GetInstance().CreateDevice(info));
         printf("Opening camera %d...\n", current_id);
 		camera->Open();
-        printf("Setting interpacket delay...\n");
-		camera->GevSCPD.SetValue(600);
+        camera->GammaSelector.SetValue(Basler_GigECamera::GammaSelector_User); //Necessary for interface to work
+        camera->AcquisitionFrameRateEnable.SetValue(true); //Turn on capped framerates
+        //let camera send timestamps and FrameCounts.
+		if(GenApi::IsWritable(camera->ChunkModeActive)){
```

Comment:
mixed tab and space indentation

Reply:
What are the correct indentation settings for this project? I tried changing it but not sure if this is better now.

Reply:
It is not consistent unfortunately... Just make sure it fits around the existing code.